src/System.Runtime.InteropServices.RuntimeInformation/src/OSPlatform.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Runtime.InteropServices
+{
+    public struct OSPlatform : IEquatable<OSPlatform>
+    {
+        private readonly string _osPlatformString;
+
+        private const string WindowsString = "WINDOWS";
+        private const string LinuxString = "LINUX";
+        private const string OSXString = "OSX";
```

Comment:
Please also add `private const string FreeBSDString = "FreeBSD";`

CoreCLR is recently configured to build on FreeBSD with CI build setup.
Some folks are already working on corefx to build against FreeBSD (search "freebsd" on GitHub in corefx and coreclr repos). With that said, I think FreeBSD deserves some love here. :)

copy @josteink, @saper, @ghuntley, @mmitche